site/data/statistics.yaml

```diff
@@ -583,6 +583,48 @@
   code: main/addOns/exim/src/main/java/org/zaproxy/addon/exim/urls/UrlsImporter.java
   desc: The number of URLs imported via a file
 
+- key: stats.formhandler.add
+  scope: global
+  type: counter
+  repo: zaproxy/zap-extensions
+  code: main/addOns/formhandler/src/main/java/org/zaproxy/zap/extension/formhandler
```

Comment:
It isn't a single class, so I just linked to the package.
<img width="599" height="273" alt="image" src="https://github.com/user-attachments/assets/eb033f94-2e41-4867-b9d1-fb9baf30b181" />